web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -34,40 +35,37 @@ interface Props {
   fireZoneAreas: FireZoneArea[]
 }
 
-const ZoneSummaryPanel = (props: Props) => {
+const ZoneSummaryPanel = React.forwardRef((props: Props, ref: any) => {
```

Comment:
I think you can use `React.ForwardedRef<HTMLDivElement>` as the type instead of `any`.

Reply:
> 

Thanks, good call